sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/implementation/OpenAIClientImpl.java

```diff
@@ -1858,17 +1606,17 @@ public Response<BinaryData> getChatCompletionsWithAzureExtensionsWithResponse(
      *     user: String (Optional)
      * }
      * }</pre>
-     *
-     * <p><strong>Response Body Schema</strong>
-     *
+     * <p>
+     * <strong>Response Body Schema</strong>
+     * </p>
      * <pre>{@code
      * {
      *     id: String (Required)
      *     created: long (Required)
      *     expires: Long (Optional)
      *     result (Optional): {
      *         created: long (Required)
-     *         data: DataModelBase (Required)
+     *         data: BinaryData (Required)
```

Comment:
Another `union` type that we will need to adjust. Just leaving a comment for our visibility and coordinate work around migrating this appropriately.

Reply:
IIRC, DataModelBase has been introduced since the last type spec emitter 0.9.0 but we want to replace it with BinaryData